sys/include/net/gnrc/netdev2.h

```diff
@@ -102,6 +103,16 @@ typedef struct gnrc_netdev2 {
      * @brief general information for the MAC protocol
      */
     uint16_t mac_info;
+
+    /**
+     * @brief device's l2 address
+     */
+    uint8_t  l2_addr[IEEE802154_LONG_ADDRESS_LEN];
```

Comment:
@miri64 

> Maybe a good idea to store the address regardless, just to save some time and code.

Really sorry for my English, but I am not sure I got the idea, so do you mean we still keep this PR?
which introduces `uint8_t  l2_addr[IEEE802154_LONG_ADDRESS_LEN]` into `gnrc_netdev2`?